streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/application/impl/ApplicationManageServiceImpl.java

```diff
@@ -463,6 +471,19 @@ public Long copy(Application appParam) {
   public boolean update(Application appParam) {
     Application application = getById(appParam.getId());
 
+    /* If the original mode is remote, k8s-session, yarn-session, check cluster status */
+    FlinkExecutionMode flinkExecutionMode = application.getFlinkExecutionMode();
+    switch (flinkExecutionMode) {
+      case REMOTE:
+      case YARN_SESSION:
+      case KUBERNETES_NATIVE_SESSION:
+        FlinkCluster flinkCluster = flinkClusterService.getById(application.getFlinkClusterId());
+        ApiAlertException.throwIfFalse(
+            flinkClusterWatcher.getClusterState(flinkCluster) == ClusterState.RUNNING,
+            "[StreamPark] update failed, because bind flink cluster not running");
```

Comment:
I think the error message here can be modified

Reply:
Do you mean it can be replaced dynamically? But it is only used to check whether the cluster is running.

Reply:
What I mean is, [StreamPark] update failed, because bind flink cluster not running, this log can be more concise

Reply:
What I mean is, [StreamPark] update failed, because bind flink cluster not running, this log can be more concise,